README.md

```diff
@@ -428,6 +428,18 @@ $ node -e "console.log(require('./index).main)"
 [Function]
 ```
 
+If you want to run the program (akin to `pulp run`), just use `run`:
+```bash
+# The main module defaults to "Main"
+$ spago run
+
+# Or define your own module path to Main
+$ spago run --main ModulePath.To.Main
+
+# And pass arguments through to your program
+$ spago run --main ModulePath.To.Main -- --some-option "argument"
```

Comment:
These additional arguments are passed to `purs compile`

Reply:
Hmm... Well, then... This command does not work "akin to `pulp run` then, AFAIK.

Is that possibility (pass args to `purs compile` and other args to the actual program) desired?

Reply:
If we want to make it work like `pulp run` then it's desired. Though I think we could take care of that in another PR?